src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1398,6 +1398,28 @@ paths:
                 $ref: '#/components/schemas/Info'
         default:
             $ref: '#/components/responses/defaultResponse'
+    delete:
+      operationId: deleteLicenseByShortname
+      tags:
+      - License
+      summary: delete a specific license
+      description: >
+        delete information about a specific license
+      responses:
+        '200':
+          description: License deleted successfully
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/License"
```

Comment:
I believe when a license is successfully deleted, you are just returning an object of type `'#/components/schemas/Info'` and not the complete license info (as in GET).

Reply:
Thanks @GMishx  for the review. I will just change it.